.github/workflows/build_and_release_binary.yaml

```diff
@@ -302,12 +278,6 @@ jobs:
         if: steps.compare_version.outputs.result != 'true'
         run: |
           $env:path += ";deps\cmake\bin\"
-          # remove warning-as-erro flag in cmake
-          sed -i '/option(PEDANTIC \"Enable extra warnings and pedantic build flags. Treat all warnings as errors.\" ON)/d' CMakeLists.txt
-          sed -i '/add_compile_options(-Werror)/d' ./cmake/EthCompilerSettings.cmake
-          sed -i '/add_compile_options(-pedantic)/d' ./cmake/EthCompilerSettings.cmake
-          sed -i '/add_compile_options(\\/WX)/d' ./cmake/EthCompilerSettings.cmake
-          sed -i '/elseif (DEFINED MSVC)/d' ./cmake/EthCompilerSettings.cmake
```

Comment:
Does anyone know why we had to remove the `elseif (DEFINED MSVC)`?

Reply:
I'm afraid we can't access that info anymore.
It is fine if it works, but I suspect lower versions will fail without it.

Reply:
Ah, I remember that we couldn't disable the pedantic flags in older versions since -DPEDANTIC was not supported.

Not sure if it's related, but we missed removing `/WX` (warnings as errors) in the MSVC configuration when we removed -Werror in the cmake for all the versions. We'll need to check the CI of an older version.

Anyways, I still don't understand why we have this line.